icommons_ext_tools/settings/base.py

```diff
@@ -301,8 +274,8 @@
     'QUALTRICS_API_USER': SECURE_SETTINGS.get('qualtrics_api_user'),
     'QUALTRICS_API_TOKEN': SECURE_SETTINGS.get('qualtrics_api_token'),
     'QUALTRICS_AUTH_GROUP': SECURE_SETTINGS.get('qualtrics_auth_group'),
-    'USER_DECLINED_TERMS_URL': 'ql:internal',
-    'USER_ACCEPTED_TERMS_URL': 'ql:internal',
+    'USER_DECLINED_TERMS_URL': SECURE_SETTINGS.get('qualtrics_user_declined_terms_url'),
```

Comment:
those values for dev and qa should be 'ql:internal' it's only for prod that they are different so maybe stage and prod they should be 'http://surveytools.harvard.edu'